compiler/rustc_codegen_cranelift/src/constant.rs

```diff
@@ -272,7 +272,7 @@ fn data_id_for_static(
             .layout_of(ty::TypingEnv::fully_monomorphized().as_query_input(ty))
             .unwrap()
             .align
-            .pref
+            .abi
```

Comment:
I can implement some code that functionally replaces this with an "alignment-padded" variant for codegen purposes if that would preferable, but I don't think it makes the codegen incorrect to simply stop.

Reply:
This is fine. cg_clif doesn't have great runtime perf anyway.

Reply:
I feel dubious about whether it even positively affects runtime.